rudder-core/src/main/scala/com/normation/rudder/batch/AsyncDeploymentAgent.scala

```diff
@@ -162,14 +157,50 @@ final class AsyncDeploymentAgent(
     )
   }
 
-  override protected def lowPriority = {
+  // That flag file is used to determine if a policy update was running when the webapp stopped so we can launch a new one when starting
+  val policyUpdateRunningFlagPath = "/opt/rudder/etc/policy-update-running"
```

Comment:
That must be a parameter, perhaps even something coming from config (with a default value if not provided), and documented. 
More over, it should go to /var/rudder, no ?